net-mgmt/zabbix-agent/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=		zabbix-agent
+PLUGIN_VERSION=		0.1
+PLUGIN_COMMENT=		Enterprise-class open source distributed monitoring agent
+PLUGIN_DEPENDS=     zabbix32-agent
```

Comment:
the package name changes, no problem until they install the plugin, but I'll try to think of something.